docs/crio.8.md

```diff
@@ -280,6 +281,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--selinux**: Enable selinux support (default: false)
 
+**--separate-pull-cgroup**: [EXPERIMENTAL] Pull in new cgroup (default: false)
```

Comment:
I think instead of boolean we could make this a string. So, we have:
1. empty string: current behavior
2. POD: pull in pod cgroup
3. some.slice: Perform all pulls in that slice